lyndapham/step/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,34 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> arr;
+  Cookie choc;
+
+  @Override
+  public void init() {
+    arr = new ArrayList<>();
+    arr.add("Chocolate chip");
+    arr.add("Oatmeal raisin");
+    arr.add("Snickerdoodle");
+
+    choc = new Cookie ("Chocolate chip", 5);
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Lynda!");
+    Gson gson = new Gson();
+    String json = gson.toJson(choc);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
   }
 }
+
+class Cookie {
```

Comment:
In general don't push any code that you are using to test functionality like you are doing here. Tests, such as unit tests should be in their own file. 

Classes defined at the top level of a class can either be public or not, in this case Cookie is not and is therefore package private meaning that it can only be referenced within the same package, i.e only by files with the same package name. For what you were trying to do, Cookie should be a private inner class.

Reply:
I removed Cookie and have added it to a personal branch!